### PR TITLE
Refactors the code that handles ARB annotations in parseResources

### PR DESCRIPTION
**Pull request by fyoshimatsu**
... in order to make it handle placeholders better later

Note that the ARB annotations are parsed into `notes` exactly the same way as the current code does. It will change later in separate pull requests so we can handle `placeholders` better, for example.

### CODE REVIEW

**Comment by diegone** [comment]
do we need to bump the version and publish to npm? if so, can you bump the version?

**Comment by fyoshimatsu** [comment]
I have two more PRs to make to the helper in mind. Is it okay to consolidate all of them in one new version?

File: regression/mint/react-icu/package.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
     "name": "i18next-demo",
     "dependencies": {
         "@l10nmonster/helpers": "^1",
-        "@l10nmonster/helpers-json": "^1"
+        "@l10nmonster/helpers-json": "../../../helpers-json"
```
Comment (fyoshimatsu) [inline regression/mint/react-icu/package.json:5]: @diegone could this be the solution for the problem we talked about, that the regression tests require npm modules to be published first?

Comment (diegone) [reply regression/mint/react-icu/package.json:5]: that will make it work locally but it won't test whether it'll break clients that use npm

Comment (fyoshimatsu) [reply regression/mint/react-icu/package.json:5]: So I guess we want to execute tests using the code in the `dist` or somewhere else that has artifacts of a build?

File: regression/expected/react-icu/TMExport_en_de_job_tmexport_en_de-req.json [hunk 2]
```diff
@@ -304,6 +304,22 @@
...
+			"guid": "6DAv7B6YywviZSG0LkH6iEmPaGfotuTy4Ljqm6eumfk",
+			"notes": {
+				"desc": "copy - national ID input placeholder on passenger form\nplaceholders: {\"id\":{\"example\":\"CPF\",\"description\":\"Name of a national ID\"}}"
+			},
+			"seq": 1006,
+			"nsrc": [
+				"Enter your ",
+				{
+					"t": "x",
+					"v": "{{id}}"
```
Comment (theresakhoo) [inline regression/expected/react-icu/TMExport_en_de_job_tmexport_en_de-req.json:319]: The placeholder samples should be reflected in an `s` entry, for example [see](https://github.com/l10nmonster/l10nmonster/blob/49ad4ee6d1ec2ccb82dd362b827ac5b03821ca3d/tests/normalizers/util.test.js#L28)

File: regression/expected/react-icu/translationJobs/en_de/Visicode_en_de_job_xxx2xxx-req.json [hunk 3]
```diff
@@ -228,6 +228,22 @@
...
+			"guid": "6DAv7B6YywviZSG0LkH6iEmPaGfotuTy4Ljqm6eumfk",
+			"notes": {
+				"desc": "copy - national ID input placeholder on passenger form\nplaceholders: {\"id\":{\"example\":\"CPF\",\"description\":\"Name of a national ID\"}}"
+			},
+			"seq": 1006,
+			"nsrc": [
+				"Enter your ",
+				{
+					"t": "x",
+					"v": "{{id}}"
```
Comment (theresakhoo) [inline regression/expected/react-icu/translationJobs/en_de/Visicode_en_de_job_xxx2xxx-req.json:243]: The sample should be added here as an `s`

**Pull request merged**